projects/1.16/assets/supplementaries/supplementaries/lang/zh_cn.json

```
@@ -1 +1,1215 @@
+  "block.supplementaries.hanging_sign_embur": "余烬焦木挂牌",
+  "block.supplementaries.hanging_sign_ether": "以太木挂牌",
+  "block.supplementaries.hanging_sign_fir": "冷杉木挂牌",
+  "block.supplementaries.hanging_sign_green_enchanted": "绿色魔法木挂牌",
+  "block.supplementaries.hanging_sign_holly": "冬青木挂牌",
+  "block.supplementaries.hanging_sign_jacaranda": "蓝花楹木挂牌",
+  "block.supplementaries.hanging_sign_lament": "哀叹木挂牌",
+  "block.supplementaries.hanging_sign_mahogany": "桃花心木挂牌",
+  "block.supplementaries.hanging_sign_mangrove": "红树林木挂牌",
+  "block.supplementaries.hanging_sign_maple": "枫木挂牌",
+  "block.supplementaries.hanging_sign_nightshade": "夜影木挂牌",
+  "block.supplementaries.hanging_sign_palm": "棕榈木挂牌",
+  "block.supplementaries.hanging_sign_palo_verde": "扁轴木挂牌",
+  "block.supplementaries.hanging_sign_pine": "松木挂牌",
+  "block.supplementaries.hanging_sign_rainbow_eucalyptus": "彩虹桉木挂牌",
+  "block.supplementaries.hanging_sign_redwood": "红杉木挂牌",
+  "block.supplementaries.hanging_sign_skyris": "天际木挂牌",
+  "block.supplementaries.hanging_sign_sythian": "斯基泰木挂牌",
+  "block.supplementaries.hanging_sign_willow": "柳木挂牌",
+  "block.supplementaries.hanging_sign_witch_hazel": "金缕梅木挂牌",
+  "block.supplementaries.hanging_sign_zelkova": "榉木挂牌",
+  "item.supplementaries.sign_post_aspen": "白杨木路标",
+  "item.supplementaries.sign_post_baobab": "猴面包木路标",
+  "item.supplementaries.sign_post_blue_enchanted": "蓝色魔法木路标",
+  "item.supplementaries.sign_post_bulbis": "灯状菇路标",
+  "item.supplementaries.sign_post_cherry": "樱桃木路标",
+  "item.supplementaries.sign_post_cika": "锡卡木路标",
+  "item.supplementaries.sign_post_cypress": "柏木路标",
+  "item.supplementaries.sign_post_ebony": "乌木路标",
+  "item.supplementaries.sign_post_embur": "余烬焦木路标",
+  "item.supplementaries.sign_post_ether": "以太木路标",
+  "item.supplementaries.sign_post_fir": "冷杉木路标",
+  "item.supplementaries.sign_post_green_enchanted": "绿色魔法木路标",
+  "item.supplementaries.sign_post_holly": "冬青木路标",
+  "item.supplementaries.sign_post_jacaranda": "蓝花楹木路标",
+  "item.supplementaries.sign_post_lament": "哀叹木路标",
+  "item.supplementaries.sign_post_mahogany": "桃花心木路标",
+  "item.supplementaries.sign_post_mangrove": "红树林木路标",
+  "item.supplementaries.sign_post_maple": "枫木路标",
+  "item.supplementaries.sign_post_nightshade": "夜影木路标",
+  "item.supplementaries.sign_post_palm": "棕榈木路标",
+  "item.supplementaries.sign_post_palo_verde": "扁轴木路标",
+  "item.supplementaries.sign_post_pine": "松木路标",
+  "item.supplementaries.sign_post_rainbow_eucalyptus": "彩虹桉木路标",
+  "item.supplementaries.sign_post_redwood": "红杉木路标",
+  "item.supplementaries.sign_post_skyris": "天际木路标",
+  "item.supplementaries.sign_post_sythian": "斯基泰木路标",
+  "item.supplementaries.sign_post_willow": "柳木路标",
+  "item.supplementaries.sign_post_witch_hazel": "金缕梅木路标",
+  "item.supplementaries.sign_post_zelkova": "榉木路标",
+  "block.supplementaries.hanging_sign_imparius": "Imparius 挂牌",//官中未译
+  "item.supplementaries.sign_post_imparius": "Imparius 路标",
+  //bop = Biomes O' Plenty

```

您可以用不存在的key来表达注释
```suggestion
  "block.supplementaries.hanging_sign_imparius": "Imparius 挂牌",
  "_supplementaries.comment.hanging_sign_imparius": "官中未译"
  "item.supplementaries.sign_post_imparius": "Imparius 路标",
  "_supplementaries.comment.bop": "Biomes O' Plenty"
```
如果支持json5规划就可以直接注释了